### PR TITLE
pin librsvg to new 2.44.14 version

### PR DESCRIPTION
**Pull request by bgruening**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by isuruf** [comment]
@bgruening, can you send a PR to add run_exports to librsvg?

**Comment by ocefpaf** [comment]
> @bgruening, can you send a PR to add run_exports to librsvg?

Done in https://github.com/conda-forge/librsvg-feedstock/pull/26

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -240,7 +240,7 @@ pin_run_as_build:
   librdkafka:
     max_pin: x.x.x
   librsvg:
-    max_pin: x.x
+    max_pin: x
```
Comment (ocefpaf) [inline recipe/conda_build_config.yaml]: I was going for a `x.x` b/c we don't really have the ABI info on version `2.43`. There is no reason to doubt that `x` is good enough but I prefer to play on the safe side though.

What do you think @isuruf?

Comment (isuruf) [reply recipe/conda_build_config.yaml]: I looked at the symbols and 2.44 added a bunch of symbols compared to 2.40 (I think in part due to the switch to rust) and 4 symbols removed.

```
$diff -ru 2.40.txt 2.44.txt  | grep -E "^\-"
--- 2.40.txt    2019-07-22 09:38:07.325532888 -0500
-rsvg_cairo_to_pixbuf
-rsvg_css_parse_color
-rsvg_pixbuf_from_data_with_size_data
```

According to https://launchpad.net/debian/+source/librsvg/+changelog these should have been private and no package in debian links to them.

Comment (ocefpaf) [reply recipe/conda_build_config.yaml]: So `x.x` then?

File: recipe/conda_build_config.yaml [hunk 2]
```diff
@@ -462,7 +462,7 @@ libprotobuf:
 librdkafka:
   - 0.11.5
 librsvg:
-  - 2.44.3
+  - 2
```
Comment (ocefpaf) [inline recipe/conda_build_config.yaml]: I guess this should be `2.44.14` to ensure we are getting the latest one with `run_exports` (after we merge https://github.com/conda-forge/librsvg-feedstock/pull/26 and remove the old build without `run_exports`.)

Comment (isuruf) [reply recipe/conda_build_config.yaml]: We'll have to keep bumping the version if it's `2.44.13`.

File: recipe/conda_build_config.yaml (hunk 1 above)
Comment (isuruf) [reply recipe/conda_build_config.yaml]: I was making the case for `x` above. :smile:

File: recipe/conda_build_config.yaml (hunk 2 above)
Comment (ocefpaf) [reply recipe/conda_build_config.yaml]: Not sure what you mean? Bump the version of what depend on it? If so wouldn't a migration be enough to take care of this problem?

File: recipe/conda_build_config.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipe/conda_build_config.yaml]: If you think it is stable enough then sure! Let's go for it!

File: recipe/conda_build_config.yaml (hunk 2 above)
Comment (isuruf) [reply recipe/conda_build_config.yaml]: We don't need a migration until version 3.0.
Say we bump `pango` which is a dependency of librsvg and `pango` was bumped in `librsvg==2.44.14` (a future version). Then we need to bump `librsvg` here to `2.44.14` as well which is unnecessary work IMO and if we had left this as `2` we don't need to do that.

**Pull request merged**